docs/team.mdx

```diff
@@ -6,11 +6,31 @@ title: Zenith Team
 The Zenith Emerging Technologies Special Interest Group (Zenith SIG) co-chairs are:
 
 <Bio name="Keith O'Donnell" organisation="Feynic" image="/img/people/keith-o-donnell.png">
+Keith O’Donnell is the Managing Director of Feynic Technology Ltd, an advisory consultancy focused on the commercialisation of emerging technologies. 
+
+He previously led the EMEA Innovation Program for Morgan Stanley, seeding their Spatial Computing & Quantum Technology programs and developing numerous proof-of-concept projects across a number of disciplines relevant to this special interest group proposal.
+
+Working across numerous industries since 2001, Keith has earned a varied skillset that enables him to adapt quickly to new technologies and disciplines. This experience covers the construction, logistics, finance and technology sectors. He currently holds an affiliate position with the Scottish International Space Advisory Council (SISAC) and is an active member of the Global Scot network.
+
+Keith earned his degree in Business from the University of Strathclyde, having previously studied Genetics & Biotechnology at the University of Glasgow.
 </Bio>
 
 <Bio name="Patrick Downing" organisation="Morgan Stanley" image="/img/people/patrick-downing.png">
+Patrick Downing is a Vice President of Identity and Access Management focusing on strategic, core infrastructure initiatives across Morgan Stanley. 
+

```

Comment:
Erroneous spacing issue

Reply:
I feel like I am blind, where?

Reply:
Seems there are a few of them between the three bios. On the preview page the paragraph/sentence spacing is different. Looks like:

![image](https://github.com/finos/zenith/assets/19316277/f33ce44e-0afc-42d6-bf99-02f9cee75bae)

Reply:
Ah, I see your point. And do you think I should merge the two paragraphs or the opposite, change it to a real new line?

Reply:
For this one I think it's fine to have as a single paragraph. Looks like it's the first two lines of each bio that have the same issue however. So would need to give them the same treatment.

Reply:
I copied them off the deck, so it might have had some special character in it